docs/_source/practical_guides/filter_dataset.md

```diff
@@ -117,6 +117,39 @@ sorted_records = remote.sort_by(
 You can also combine filters and sorting: `dataset.filter_by(...).sort_by(...)`
 ```
 
+### Semantic search
+You can retrieve records based on their similarity with another record. To do that, make sure you have added `vector_settings` to your [dataset configuration](create_dataset.md#define-vectors) and that your [records include vectors](create_dataset.md#configure-the-records).
+
+In the UI, go to the record you'd like to use for the semantic search and click on `Find similar` at the top right corner of the record card. If there is more than one vector, you will be asked to select which vector to use. You can also select whether you want the most or least similar records and the number of results you would like to see.
+
+In the Python SDK, you can also get a list of records that are semantically close to a given embedding with the `find_similar_records` method. These are the arguments of this function:
+
+- `vector_name`: The `name` of the vector to use in the search.
+- `value`: A vector to use for the similarity search in the form of a `List[float]`. It is necessary to include a `value` **or** a `record`.
+- `record`: A `FeedbackRecord` to use as part of the search. It is necessary to include a `value` **or** a `record`.
+- `max_results` (optional): The maximum number of results for this search. The default is `50`.
+
+```python
+ds = rg.FeedbackDataset.from_argilla("my_dataset", workspace="my_workspace")
+
+# using text embeddings
+similar_records =  ds.find_similar_records(
+    vector_name="my_vector",
+    value=embedder_model.embeddings("My text is here")
+)
+
+# using another record
+similar_records =  ds.find_similar_records(
```

Comment:
Here `similar_records` is a list of Tuple, including the record and the similarity score (between 0 and 1).

Maybe a loop for can help users to understand how the daata ashould be processed:
````python
for record, score in similar_records:
   ...
````